controllers/sessionController.js

```diff
@@ -1,8 +1,10 @@
 const User = require("../models/User");
 const parseVErr = require("../utils/parseValidationErrs");
+const csrf = require('csurf');
+const csrfProtection = csrf();
```

Comment:
Requires change:
This csrfProtection should not be required in the controller, instead it should reside within the **route** before the controllers.